tests/providers/amazon/aws/triggers/test_eks.py

```diff
@@ -453,3 +455,403 @@ async def test_eks_nodegroup_trigger_run_attempts_failed(self, mock_async_conn,
 
         assert "Error checking nodegroup" in str(exc.value)
         assert a_mock.get_waiter().wait.call_count == 3
+
+
+class TestEksCreateClusterTrigger:
+    def test_eks_create_cluster_trigger_serialize(self):
```

Comment:
As we already wrap these test  cases as a class, not sure whether we still need the `eks_create_cluster_trigger` prefix

```suggestion
    def test_serialize(self):
```

Reply:
You're right that the test is part of the class, but I think being able to figure out the what the test is testing just from the name is fairly useful. But I'm not sure what the standard here is. I'm open to changing the names as well. What do you think?